decentraland/unity-renderer
# fix: sign up UI

## What does this PR change?

This PR fixes the missing elliptical mask of the avatar snapshot and adds a texture to the scrollbar sliding area in order to make it stand out.

Mask before vs. after
<img width="274" alt="Screen Shot 2022-04-22 at 16 55 54" src="https://user-images.githubusercontent.com/51088292/173434158-8ff601b1-7a50-4d97-8977-f06a18c9c99a.png">
<img width="264" alt="Screen Shot 2022-06-14 at 16 21 20" src="https://user-images.githubusercontent.com/51088292/173671450-f69897d2-3322-4c37-a7cc-ce744b59e0f6.png">


Scrollbar sliding area before vs. after
<img width="832" alt="Screen Shot 2022-06-14 at 16 21 02" src="https://user-images.githubusercontent.com/51088292/173671481-bd209e2e-c9a8-44a8-88ff-ccd7bd7c4ee6.png">
<img width="849" alt="Screen Shot 2022-06-14 at 16 19 53" src="https://user-images.githubusercontent.com/51088292/173671489-2f124e91-1105-44f1-8e37-49c79cb4b25f.png">


## How to test the changes?

1. Create a new account from the following link ( to make it easier, enter in incognito mode with Chrome and select "Play as Guest"): https://play.decentraland.zone/?renderer-branch=fix/SignupHUD-UI
2. After creating your avatar you should see that your profile picture is masked inside an ellipse.
3. In the step Terms of Service (...) you should see the extension of the sliding area, not just the handle.
